R/experimentSpec.R

```diff
@@ -206,13 +207,14 @@ h_gene_data <- function(object, name_annotation) {
 #'   my_app()
 #' }
 experimentSpecServer <- function(id,
-                                 datasets,
+                                 data,
+                                 filter_panel_api,
                                  mae_name,
                                  name_annotation = "symbol",
                                  sample_vars_as_factors = TRUE,
                                  with_mae_col_data = TRUE) {
   assert_string(id)
-  assert_r6(datasets)
+  assert_list(data)
```

Comment:
Is there a way to make this more specific maybe? Otherwise ok as proposed

Reply:
yeah this PR is not done yet, in progress

Reply:
I have created an issue for the deep tests failures: https://github.com/insightsengineering/teal.modules.hermes/issues/279